test/e2e/sourcery.js

```diff
@@ -41,9 +92,20 @@ module.exports = {
                 .waitForElementVisible('.user-info__title');
             }
         });
-        //Assert if expected user is logged in
-        browser.assert.containsText('.user-info__title', 'Demo User')
-            .saveScreenshot(conf.imgpath(browser) + 'Demo.png')
-            .end();
+
+        //Assert if user name is displayed
+        browser.assert.containsText('.user-info__title', 'Marija Janeckaite');
+
+        browser.assert.containsText('.main-nav', 'Time Logging');
+        browser.assert.containsText('.main-nav', 'Invoices');
+        browser.assert.containsText('.main-nav', 'Tasks');
+        browser.assert.containsText('.main-nav', 'Projects');
+        browser.assert.containsText('.main-nav', 'Clients');
+        browser.assert.containsText('.main-nav', 'Time Entries');
```

Comment:
Not mandatory but you could get rid of "browser" in every line of code so readability is a bit better.